10-open.md

```diff
@@ -104,3 +104,22 @@ by acting as a shareable electronic lab notebook for computational work:
 > using only what they can find in your papers or on the web.
 > Try to do the same for one of their results,
 > then try to do it for a result from a lab you work with.
+
+> ## How to find an appropriate data repository? {.challenge}
+>
+> Surf the internet for a couple of minutes and check out the data repositories
+> mentioned above: [Figshare](http://figshare.com/), [Zenodo](http://zenodo.org), 
+> [Dryad](http://datadryad.org/). Depending on your field of research, you might 
+> find community-recognized repositories that are well-known in your field.
+> You might also find this page very useful -> [Nature: Recommended Data
```

Comment:
Dear Ivan,

 

Thanks for your suggestions, I changed the challenges accordingly. Should I make a new pull request?

 

Best wishes,
Matthias

 

From: Ivan Gonzalez [mailto:notifications@github.com] 
Sent: Sonntag, 6. März 2016 16:03
To: swcarpentry/git-novice
Cc: Matthias
Subject: Re: [git-novice] Add two challenges to 10-open.md (#241)

 

In 10-open.md <https://github.com/swcarpentry/git-novice/pull/241#discussion_r55141481> :

> @@ -104,3 +104,22 @@ by acting as a shareable electronic lab notebook for computational work:
>  > using only what they can find in your papers or on the web.
>  > Try to do the same for one of their results,
>  > then try to do it for a result from a lab you work with.
> +
> +> ## How to find an appropriate data repository? {.challenge}
> +>
> +> Surf the internet for a couple of minutes and check out the data repositories
> +> mentioned above: [Figshare](http://figshare.com/), [Zenodo](http://zenodo.org), 
> +> [Dryad](http://datadryad.org/). Depending on your field of research, you might 
> +> find community-recognized repositories that are well-known in your field.
> +> You might also find this page very useful -> [Nature: Recommended Data

Could you incorporate the link into the sentence? Like "You might find useful these data repositories recommended by Nature"

—
Reply to this email directly or view it on GitHub <https://github.com/swcarpentry/git-novice/pull/241/files#r55141481> .  <https://github.com/notifications/beacon/AKDjif71TBLwiYp-eV7sUibO4c9ExDJNks5pquyXgaJpZM4Hf-xG.gif>